src/stories/components/HorizontalTitle/styles.scss

```diff
@@ -0,0 +1,45 @@
+@import '../../../scss/variables.scss';
+
+// @file
+// HorizontalTitle
+
+.horizontal-title {
+    font-family: 'Work Sans';
+    padding-top: 0;
+    margin-top: 0;
+
+    &--medium {
+        .horizontal-title__title {
+            font-size: 20px;
+        }
+    }
+    &--large {
+        .horizontal-title__title {
+            font-size: 24px;
+        }
+    }
+}
+
+.horizontal-title__title {
+    font-size: 16px;
+    padding-top: 0;
+    margin-top: 0;
+    padding-bottom: 10px;
+    margin-bottom: 30px;
+}
+
+.horizontal-title__eyebrow {
+    font-family: 'Work Sans';
+    font-size: 13px;
+    text-transform: uppercase;
+    letter-spacing: 1px;
+    padding-bottom: 0px;
+    margin: 0px;
```

Comment:
1. Font family is being applied to `body` and is inherited by all of its children, so no need to declare it
2. Convert font-size to `rem` by doing `rem(13px)`
3. No need to declare margin and padding, the `div` has no padding nor margins by default